apps/testnorge-skd/skd-core/src/main/java/no/nav/registre/skd/service/SyntetiseringService.java

```diff
@@ -131,16 +129,14 @@
                 }
             } catch (ManglendeInfoITpsException e) {
                 httpStatus = loggExceptionOgLeggTilFeiletEndringskode(e,
-                        "ManglendeInfoITPSException på endringskode " + endringskode.getEndringskode() + AVSPILLERGRUPPE + genereringsOrdreRequest.getAvspillergruppeId() +
-                                SKDMELDING_ID_STRING + lagGrupperAvIder(idsLagretITpsfMenIkkeTps)
-                                + SKDMELDINGER_TPSF_STRING + lagGrupperAvIder(ids),
-                        endringskode.getEndringskode());
+                        "ManglendeInfoITPSException på endringskode " + String.format(FEILMELDING, endringskode.getEndringskode(),
```

Comment:
Bra! Eneste nå er at det vel kommer ut "ManglendeInfoITPSException på endringskode Feil på endringskode ...", altså endringskode to ganger, så bare ha 

`"ManglendeInfoITPSException: " + String.format(FEILMELDING, ...);` isteden :D Samme på de to under her også 👍

Reply:
Oj, den catcha jeg ikke nei 😅👍 Skal fikse :D